Task 3/Calculator/src/Calculator.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Write first number");
+        double firstNumber = sc.nextDouble();
+        System.out.println("Choose operation: +, -, *, /, % ");
+        String operation = sc.next();
+        System.out.println("Write second number");
+        double secondNumber = sc.nextDouble();
+        System.out.print("Result=");
```

Comment:
It should be at the end, as  if we go to the incorrect operator, then we will get: Result = Something goes wrong, try again

Reply:
I didn`t understand what should be at the end? The end of what? As for me, it works like it should. Could you describe the problem more precisely?